Keas.Mvc/Handlers/VerifyRoleAccessHandler.cs

```diff
@@ -28,18 +28,28 @@ public VerifyRoleAccessHandler(ApplicationDbContext dbContext, ISecurityService
             string team = "";
```

Comment:
@jcubbage this is fine but I prefer `string.empty`

Reply:
So that role doesn't show up on the TeamRole list. We don't want systemAdmins being added to teams.

Reply:
@jcubbage Ah that makes sense